pyteal/ast/acct.py

```diff
@@ -0,0 +1,59 @@
+from typing import TYPE_CHECKING
+
+from ..types import TealType, require_type
+from ..ir import Op
+from .expr import Expr
+from .maybe import MaybeValue
+
+if TYPE_CHECKING:
+    from ..compiler import CompileOptions
+
+
+class AccountParam:
+    @classmethod
+    def balance(cls, acct: Expr) -> MaybeValue:
+        """Get the current balance in microalgos an account.
+
+        Args:
+            acct: An index into Txn.accounts that corresponds to the application to check or an address in the Txn.accounts array.
+                May evaluate to uint64 or an address.
+        """
```

Comment:
nit: I think you might still want to use `require_type` here: e.g., https://github.com/algorand/pyteal/blob/master/pyteal/ast/app.py#L98, where we can require `anyType`.

Same applies to other class methods.

Reply:
ah good call